src/OrchardCore.Modules/OrchardCore.Settings/Views/Items/SiteSettingsDeploymentStep.Fields.Edit.cshtml

```diff
@@ -3,6 +3,7 @@
 @using Newtonsoft.Json
 @{
     var settings = Model.Settings;
+    var ignoredPropeties = new HashSet<string> { "Identifier", "Properties" };
```

Comment:
ignoredPropeties => ignoredProperties

Reply:
and that's why I shouldn't write pr's while trying to talk to y'all at triage